packages/ui/react/src/components/Forms/Input/Input.stories.tsx

```diff
@@ -128,5 +129,27 @@ const Number: Story = {
   },
 }
 
-export default meta
-export { Default, Number, Error }
+const WithAutoFocusOnError: Story = {
+  args: {
+    name: 'name',
+    type: 'text',
+    placeholder: 'This input will auto-focus when error occurs',
+    labelProps: {
+      value: 'Auto-focus on error',
+      helpValue: 'This input will automatically focus when an error occurs',
+    },
+    error: {
+      message: 'This field has an error',
+    },
+    autoFocusOnError: true,
+  },
+  parameters: {
+    controls: {
+      exclude: [
+        'publicStyle',
+      ],
+    },
+  },
+}
+
+export { Default, Error, Number, WithAutoFocusOnError, meta as default }
```

Comment:
```suggestion
export default meta
export { Default, Error, Number, WithAutoFocusOnError }
```

Could you revert the default export?

The current code fails to index the Input component in storybook

![image](https://github.com/user-attachments/assets/586c4098-f072-4b56-8948-3545ba897782)